### PR TITLE
docs: add docs for startClient option

### PR DESCRIPTION
**Pull request by FredrikOseberg**
Documents the newly added startClient option. Merge when released.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,19 @@ ReactDOM.render(
 );
 ```
 
+Defer starting of the client:
```
Comment (thomasheartman) [inline README.md:61]:
```suggestion
To defer starting the client:
```

When you defer it, how do you start the client later? Rerender with the `startClient` prop set to `true` or is there another way?

**Review by thomasheartman — APPROVED** [review]
Looks good! 🎉  Just one question that I'd like to have an answer to (to call it out as clearly as possible).

File: README.md [hunk 2]
```diff
@@ -58,6 +58,19 @@ ReactDOM.render(
 );
 ```
 
+Defer starting the polling of the client (this allows you more fine grained control over when the client should fetch the feature flag configuration, for instance if you need context values that you are fetching from the server): 
+
```
Comment (thomasheartman) [inline README.md:62]: Sorry, I'm not entirely sure I understand this bit:

> Defer starting the polling of the client

Does that mean that the provider doesn't start polling the client or that the client doesn't start polling the proxy?

I'd suggest something like the below, but I need some more input: how do you start the client later? Do you re-render with `startClient` set to `true` or can you access the client via a hook or something and start it manually, e.g (in verrrry pseudocode):

```js
const client = useProviderClient()

client.start()
```

```suggestion
You can avoid starting the client immediately on render by using these `FlagProvider` props:
- pass a client as `unleashClient` 
- set `startClient` to `false`

To start the client,
// ... do something?
```

Comment (jtaaa) [reply README.md:62]: You can start the unleash-proxy-client-js client later without interacting with the flag provider
```ts
const client = new UnleashClient({ /* ... */ })

useEffect(() => {
  const asyncProcess = async () => {
    // async work ...
    client.start()
  }
  asyncProcess()
, [])

// Pass client as `unleashClient` and set `startClient` to `false`
return <FlagProvider unleashClient={client} startClient={false} />
```

Comment (jtaaa) [reply README.md:62]: > Does that mean that the provider doesn't start polling the client or that the client doesn't start polling the proxy?

Client doesn't start polling the proxy :)

Comment (FredrikOseberg) [reply README.md]: @jtaaa 

Sorry I wasn't able to properly test and release an official version before I left on my honeymoon. I was able to release `1.1.0-beta.0` now, so you should be able to install it and verify that everything works as expected. Please let me know if something is not working correctly. I'll be back next week and look into creating a proper release including this fix.

**Pull request merged**